eth_tester/backends/pyevm/main.py

```diff
@@ -366,6 +369,14 @@ def _get_normalized_and_signed_evm_transaction(self, transaction):
         signed_evm_transaction = evm_transaction.as_signed_transaction(signing_key)
         return signed_evm_transaction
 
+    def send_raw_transaction(self, raw_transaction_hex):
+        import rlp
```

Comment:
Is there a reason this import isn't at the top level?  If there is can you add a comment so that it's clear why.

Reply:
No, previously I don't want this import affecting the original codebase, where `rlp` is not used. But you're right it should be at the top level. Thank you I will fix it.